src/routes/group.route.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
+import GroupsController from '@controllers/groupe.controller';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { Routes } from '@interfaces/routes.interface';
+import validationMiddleware from '@middlewares/validation.middleware';
+
+class GroupsRoute implements Routes {
+  public path = '/groups';
+  public router = Router();
+  public groupsController = new GroupsController();
+
+  constructor() {
+    this.initializeRoutes();
+  }
+
+  private initializeRoutes() {
```

Comment:
Implement security measures, if needed, such as authentication and authorization to protect sensitive endpoints and data. This could involve using middleware for authentication checks and ensuring that only authorized users can access certain routes.